perception/navigator_perception/nodes/shape_identification/GrayscaleContour/GrayscaleContour.cpp

```diff
@@ -313,7 +312,7 @@ bool GrayscaleContour::filterArea(std::vector<Point> contour) {
 }
 Point GrayscaleContour::findCenter(std::vector<Point>& points) {
   double x = 0, y = 0;
-  for (int i = 0; i < points.size(); i++) {
+  for (unsigned int i = 0; i < points.size(); i++) {
```

Comment:
If you want something shorter that also avoids type mismatch warnings, use size_t.